test/AutoRest.Shared.Tests/AutoRest.Shared.Tests.csproj

```diff
@@ -1,9 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net6.0</TargetFrameworks>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
What's the issue with net 6.0?

Reply:
Got error when open the sln in vs:
Error occurred while restoring NuGet packages: Invalid restore input. Invalid target framework 'unsupported'.

Reply:
It's happened in vs2019, vs2022 is ok, so do we need to compatible with vs2019?

Reply:
Let's have one target for now.